lib/utils.js

```diff
@@ -33,7 +33,8 @@ function debrack (s) {
 }
 
 function uriToFilename (uri, base) {
-  var filename = path.join(base, uri)
+  var decoded = uri.split('/').map(decodeURIComponent).join('/')
```

Comment:
When a file is uploaded, any filename is URI-encoded to form the URI.   
If the file  "hello world.ttl" exists then the resource "hello%20world.ttl" exists already too.
If there is a file on disk called  "hello%20world.ttl" then there is corresponding LDP resource "hello%2520world.ttl"